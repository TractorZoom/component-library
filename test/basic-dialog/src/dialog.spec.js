import BasicDialog from '../../../packages/basic-dialog/src';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Basic Dialog', () => {
    it('should render with required props when closed', () => {
        // given
        const mockData = {
            close: jest.fn(),
            isOpen: false,
            title: 'some dialog title',
        };

        // when
        const component = renderer.create(<BasicDialog {...mockData} />);
        let tree = component.toJSON();

        // then
        expect(tree).toMatchSnapshot();
    });
});
