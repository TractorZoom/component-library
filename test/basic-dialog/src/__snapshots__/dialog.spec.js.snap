// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Dialog should render with required props when closed 1`] = `null`;
